### PR TITLE
Portuguese Translation iOS

### PR DESCRIPTION
**Pull request by SevioCorrea**

### CODE REVIEW

File: TreeTracker/Resources/pt-BR.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+"Home.InfoLabel.TreesUploadedLabel.Text" = "Árvores carregadas";
+"Home.LogoutButton.Title" = "Alterar usuário";
+"Home.AddTreeButton.Title" = "ADICIONAR ÁRVORE";
+"Home.UploadTreesButton.Title.Start" = "Iniciar upload";
+"Home.UploadTreesButton.Title.Stop" = "Parar upload";
+"Home.MyTreesButton.Title" = "MINHAS ÁRVORES";
+
+//Add Tree Screen
+"AddTree.Title" = "Adicionar árvore";
+"AddTree.PhotoButton.Title.TakePhoto" = "Torar foto";
```
Comment (viniciusyr) [inline TreeTracker/Resources/pt-BR.lproj/Localizable.strings]: The correct is "Tirar foto". Just a wrong typed

File: TreeTracker/Resources/pt-BR.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+//Selfie Screen
+"Selfie.Title" = "Tirar foto";
+"Selfie.SaveButton.Title" = "Salvar";
+"Selfie.PhotoButton.Title.TakePhoto" = "Tirar foto";
+"Selfie.PhotoButton.Title.Retake" = "Tirar novamente";
+"Selfie.LibraryButton.Title.LibraryPhoto" = "Da galeria";
+
+//Home Screen
+"Home.Title" = "Treetracker";
+"Home.InfoLabel.TreesPlantedLabel.Text" = "Árvoes plantadas";
```
Comment (viniciusyr) [inline TreeTracker/Resources/pt-BR.lproj/Localizable.strings]: Is missing a single letter R. Just a wrong typed

File: TreeTracker/Resources/pt-BR.lproj/Localizable.strings [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+"GPSAccuracyLabel.Text.Unknown" = "Desconhecido";
+
+//Profile screen
+"Profile.FallbackTitle" = "Meu Perfil";
+"Profile.HeaderLabel.Name" = "Nome";
+"Profile.HeaderLabel.Email" = "Endereço de E-mail";
+"Profile.HeaderLabel.Phone" = "Número de Telefone";
+"Profile.HeaderLabel.Username" = "Nome de usuário";
+"Profile.HeaderLabel.Organization" = "Organização";
+"Profile.ChangeUserButton.Title" = "Mudar usuário";
```
Comment (viniciusyr) [inline TreeTracker/Resources/pt-BR.lproj/Localizable.strings]: "Trocar de usuário" or "Trocar usuário" sounds better and more native than "Mudar usuário"

File: TreeTracker/Resources/pt-BR.lproj/Localizable.strings [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+//Sign In Screen
+"SignIn.Title" = "Login";
+"SignIn.TextInput.PhoneNumber.Placeholder" = "Número do Telefone";
+"SignIn.OrLabel.Text" = "ou";
+"SignIn.TextInput.Email.Placeholder" = "E-mail";
+"SignIn.LoginButton.Title" = "Login";
+
+// Sign Up Screen
+"SignUp.Title" = "Cadastrar";
+"SignUp.TextInput.FirstName.Placeholder" = "Primeiro nome";
```
Comment (viniciusyr) [inline TreeTracker/Resources/pt-BR.lproj/Localizable.strings]: Only "Nome" sounds better and more native than "Primeiro nome".

File: TreeTracker/Resources/pt-BR.lproj/Localizable.strings [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+"SignUp.TextInput.LastName.Placeholder" = "Último nome";
```
Comment (viniciusyr) [inline TreeTracker/Resources/pt-BR.lproj/Localizable.strings]: "Sobrenome" sounds better and more native than "Último nome".

**Review by viniciusyr — CHANGES_REQUESTED** [review]
I put a comment on five of the lines where they needed a change or where a single letter is missing. After the fix, the request looks perfect.

**Comment by SevioCorrea** [comment]
> I put a comment on five of the lines where they needed a change or where a single letter is missing. After the fix, the request looks perfect.

Hi Vinicius. Thanks. I changed all the words you suggested.

**Review by viniciusyr — DISMISSED** [review]
This translation is perfect! Good job, Sevio!

File: TreeTracker.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (AlexCornforth) [inline TreeTracker.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings]: This looks unrelated to the change. I assume this is SwiftUI previews, as we're not using swiftUI i think we should revert this change

**Review by AlexCornforth — CHANGES_REQUESTED** [review]
Thanks for this @SevioCorrea 👍 

Translation look great but there are a bunch of unrelated files in here. Could you revert anything that isnt required for the new translations.

**Comment by AlexCornforth** [comment]
Closing as this was done in a different PR

**Pull request closed without merging**